src/qumin/representations/segments.py

```diff
@@ -37,16 +37,20 @@ class Form(str):
 
     Sounds might be more than one character long.
     Forms are strings, they are segmented at the object creation.
-    They have a tokens attribute, which is a tuple of phonemes.
+
+    Attributes:
+        tokens (Tuple): Tuple of phonemes contained in this form
+        id (str): form_id of the corresponding form according to the Paralex package
     """
 
-    def __new__(cls, string):
+    def __new__(cls, string, fid):
```

Comment:
Variable name: let's do `form_id`, it's longer, but more transparent and corresponds to the paralex name.

Moreover I think this might need to be an optional, keyword argument, eg `form_id=None`, as `from_segmented_str`, which is used in patterns application (.apply()), generates Forms which don't have ids.